spring/src/main/java/org/axonframework/spring/config/TransactionManagerFactoryBean.java

```diff
@@ -29,6 +29,7 @@
  * @author Allard Buijze
  * @since 2.0
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.